tests/SqlFormatterTest.php

```diff
@@ -113,6 +112,25 @@ function testCacheStats() {
 		$this->assertGreaterThan(1,$stats['hits']);
 	}
 
+	public function testSynapseTempTables() {
```

Comment:
Teda uz sa to bude dalo pouzit v `synapse-transformations` ... a vypnut `#` ako znak pre comment -> lebo to je v Synapse temp tabulka.